### PR TITLE
Update itsdevdeepak.json

### PR DESCRIPTION
**Pull request by itsdevdeepak**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7424"></a>

### CODE REVIEW

File: data/itsdevdeepak.json [hunk 1]
```diff
@@ -1,19 +1,41 @@
 {
   "name": "Deepak",
-  "bio": "Fullstack Developer | Javascript/Typescript | Open Source",
+  "bio": "Web Alchemist 🧙‍♂️ | Crafting Digital Elixirs with JavaScript  🪄 | Exploring the Open Source Realm 💫",
+  "tags": ["Open Source", "Javascript", "Typescript", "Reactjs", "NextJs", "Nodejs"]
```
Comment (kumarsonsoff3) [inline data/itsdevdeepak.json:4]: Please add the missing comma here. 

```suggestion
  "tags": ["Open Source", "Javascript", "Typescript", "Reactjs", "NextJs", "Nodejs"],
```

File: data/itsdevdeepak.json [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+      "url": "https://www.instagram.com/itsdevdeepak"
+    },
+    {
+      "icon": "FaLink",
+      "url": "https://itsdevdeepak.me"
+    },
+    {
+      "icon": "FaDollarSign",
+      "url": "https://github.com/sponsors/itsdevdeepak"
+    }
```
Comment (kumarsonsoff3) [inline data/itsdevdeepak.json:25]: Also the missing comma and bracket here. 

```suggestion
    }],
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/itsdevdeepak.json (hunk 2 above)
Comment (itsdevdeepak) [reply data/itsdevdeepak.json]: done

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**